bin/bundler-search

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Search your bundle for the provided pattern
+#   Requires bundler 1.8+ for execution as a bundler subcommand.
+#   Examples:
+#     bundle search Kernal.warn
+#     bundle search current_user clearance
+#
+# Arguments:
+#  1. What to search for
+#  2. Which gem names to search (defaults to all gems)
+ag "$1" $(bundle show --paths $2)
```

Comment:
The unquoted `$2` only works by coincidence because gems names rarely (never?) contain spaces. I would do the following anyway:

```sh
pattern="$1"; shift
ag "$pattern" $(bundle show --paths "$@")
```

It still relies on the implicit word-splitting of `$(bundle show ...)`, but there's not a great (read: POSIX) way around that. It'll break if `bundle show` reports directories with spaces.